src/components/modal/Modal.svelte

```diff
@@ -0,0 +1,408 @@
+<svelte:head>
+  {#if !scrollableBody}
+    <style>
+      body {
+        overflow: hidden;
+      }
+    </style>
+  {/if}
+</svelte:head>
+
+{#if localVisible}
+  <div class="ant-modal-root" bind:this="{thisModal}">
```

Comment:
I am wondering if we should use the [config-provider](https://ant.design/components/config-provider/) for the class prefix like i did in the [button](https://github.com/svelting/svant/blob/master/src/components/button/Button.svelte#L86), ant design uses this to specify the `ant-` prefix and it might be good for custom themes and so on to make that available globaly.

Other things that it could be used for is for `rtl` and other stuff like local and so on.


E.x the OK/Cancel button would depend on the [local ](https://github.com/ant-design/ant-design/blob/master/components/locale/default.tsx#L30).

I think stuff like this we should keep since it is super friendly especially for larger apps.

Reply:
You would need to copy and the locals and use the default local for now and make it possible to override the local by overriding the context (I already show an example in the button how to get the configs from the context or the default if it does not exist 😄)

Reply:
Oh yeah for sure. I totally overlooked the benefits. I'm glad you're bringing this up now. I'll add that.

Reply:
Wow, I'm looking into this more and it is really cool. I have to keep the ConfigProvider in mind with all future components.